docs/src/cli/transfer-tokens.md

```diff
@@ -0,0 +1,98 @@
+
+### Using Solana CLI
+
+Before running any Solana CLI commands, let's go over some conventions that
+you will see across all commands. First, the Solana CLI is actually a collection
+of different commands for each action you might want to take. You can view the list
+of all possible commands by running:
+
+```bash
+solana --help
+```
+
+To zoom in on how to use a particular command, run:
+
+```bash
+solana <COMMAND> --help
+```
+
+where you replace the text `<COMMAND>` with the name of the command you want
+to learn more about.
+
+The command's usage message will typically contain words such as `<NUMBER>`,
+`<PUBKEY>` or `<KEYPAIR>`. Each word is a placeholder for the *type* of text
+you can execute the command with. For example, you can replace `<NUMBER>`
+with a number such as `42` or `100.42`. You can replace `<PUBKEY>` with the
+base58 encoding of your public key. For `<KEYPAIR>`, it depends on what type
+of wallet you chose. If you chose an fs wallet, that path might be
+`~/my-solana-wallet/my-keypair.json`.  If you chose a paper wallet, use the
+keyword `ASK`, and the Solana CLI will prompt you for your seed phrase. If
+you chose a hardware wallet, use your USB URL, such as `usb://ledger?key=0`.
+
+### Test-drive your Public Keys
+
+Before sharing your public key with others, you may want to first ensure the
+key is valid and that you indeed hold the corresponding private key.
+
+Try and *airdrop* yourself some play tokens on the developer testnet, called
+Devnet:
+
+```bash
+solana airdrop 10 <PUBKEY> --url http://devnet.solana.com
+```
+
+where you replace the text `<PUBKEY>` with your base58 public key.
+
+If the airdrop was successful, it will output the account's balance,
+10 SOL. If you run the same command again, it should output 20 SOL.
+
+Next, proof that you own those tokens by transferring them. The Solana cluster
+will only accept the transfer if you sign the transaction with the private
+key corresponding to the sender's public key in the transaction.
+
+```bash
+solana-keygen new -o rando-keypair.json
+solana transfer --keypair=<KEYPAIR> rando-keypair.json 5
+```
+
+where you replace `<KEYPAIR>` with the path to a keypair in your wallet.
+
+If the transaction was accepted by the cluster, the command will output the
+transaction signature. The signature uniquely identifies your transaction.
+To confirm the transaction executed successfully, run:
+
+```bash
+solana confirm <SIGNATURE>
+```
+
+If it outputs "Confirmed", you can confidently share your public key and know
+that you will have control over any tokens sent to it.
+
+## Send Tokens
+
+If you already hold SOL and want to send tokens to someone, you will need
+a path to your keypair, their base58-encoded public key, and a number of
+tokens to transfer. Once you have that collected, you can transfer tokens
+with the `solana transfer` command:
+
+```bash
+solana transfer --keypair=<KEYPAIR> <PUBKEY> <NUMBER>
+```
+
+Confirm the updated balances with `solana balance`:
```

Comment:
Oh. :blush: 

Perhaps a separate section about checking balance after the `airdrop` command rather than (in addition to?) pointing out that it displays the balance